spec/features/step_by_step_navigation_spec.rb

```diff
@@ -20,12 +23,71 @@
       then_I_visit_the_navigation_steps_page_again
       and_I_see_my_selected_preferences
     end
+
+    scenario "User schedules publishing" do
+      given_there_is_a_draft_step_by_step_page
+      and_I_visit_the_scheduling_page
+      and_I_fill_in_the_scheduling_form
+      when_I_submit_the_form
+      then_I_should_see "has been scheduled to publish"
+      and_the_step_by_step_should_have_the_status "Scheduled"
+    end
+
+    scenario "User tries to schedule publishing for date in the past" do
+      given_there_is_a_draft_step_by_step_page
+      and_I_visit_the_scheduling_page
+      and_I_fill_in_the_scheduling_form_with_a_date_in_the_past
+      when_I_submit_the_form
+      then_I_should_see "Scheduled at can't be in the past"
+      and_the_step_by_step_should_have_the_status "Draft"
+    end
+
+    scenario "User tries to schedule publishing for an already scheduled step by step" do
+      given_there_is_a_scheduled_step_by_step_page
+      when_I_visit_the_scheduling_page
+      then_I_should_see "Scheduled to be published at"
+      and_there_should_be_no_schedule_button
+    end
   end
 
   def and_I_visit_the_navigation_rules_page
     visit step_by_step_page_navigation_rules_path(@step_by_step_page)
   end
 
+  def and_I_visit_the_scheduling_page
+    visit step_by_step_page_schedule_path(@step_by_step_page)
+  end
+
+  def when_I_visit_the_scheduling_page
+    # ^ reads better in some cases
+    and_I_visit_the_scheduling_page
+  end
+
+  def and_I_fill_in_the_scheduling_form
+    fill_in 'scheduled_at', with: '2030-04-20 10:26:51 UTC'
+  end
+
+  def and_I_fill_in_the_scheduling_form_with_a_date_in_the_past
+    fill_in 'scheduled_at', with: '1937-04-20 10:26:51 UTC'
+  end
+
+  def when_I_submit_the_form
+    click_on 'Schedule to publish'
+  end
+
+  def and_there_should_be_no_schedule_button
+    expect(page).not_to have_content 'Schedule to publish'
```

Comment:
Nitpick: You can check that the button itself doesn't exist. 
e.g. expect(page).not_to have_css("button", text: "Schedule to publish")